src/components/api-request.js

```diff
@@ -1150,7 +1150,7 @@ export default class ApiRequest extends LitElement {
     const queryParamObjTypeEls = [...requestPanelEl.querySelectorAll("[data-ptype='query-object']")];
     const headerParamEls = [...requestPanelEl.querySelectorAll("[data-ptype='header']")];
     const requestBodyContainerEl = requestPanelEl.querySelector('.request-body-container');
-    fetchUrl = this.path;
+    fetchUrl = this.path.replaceAll(' ', '');
```

Comment:
`space` is a valid character in url paths. they shouldn't be modified or removed

Reply:
can you provide an example where you have URL with useful spaces

Reply:
URLs are defined in [RFC 3986](http://www.ietf.org/rfc/rfc3986.txt) . Which states that spaces and few other control characters should be percentage encoded. In case of RapiDoc this is automatically done by the browser. 
Further to this, RapiDoc should not be in the business of modifying the APIs in anyways. If the spec has a mistake they must be corrected within the spec